requirements.txt

```diff
@@ -0,0 +1,7 @@
+joblib==1.4.2
+numpy==2.1.3
+pandas==2.2.3
+pycountry==24.6.1
```

Comment:
Do we use this?

Reply:
> Do we use this, pycountry?

We use pycountry in the preprocessing step of the eBible corpus to map the ISO 639-3 codes in file names to the full language names.

Reply:
Oh, that will not be used in the published package right... I think we can keep it in a separate file called dev-requirements.txt and not in the file then.